covid19lib/data.py

```diff
@@ -80,9 +93,56 @@ def preprocess_data(filename,
 
     Path(outputdir).mkdir(parents=True, exist_ok=True)
 
+    print(f"Load and process {filename}")
     df = load_preprocess_df(os.path.join(inputdir, filename+'.csv'))
-    df = add_features(df)
-    output_filepath = os.path.join(outputdir, filename+'.pkl')
-    df.to_pickle(output_filepath)
+    print(f"Add departement and region names")
+    df = add_geographical_info(df)
+    
+    output_filepath = os.path.join(outputdir, filename+'.csv.gz')
+    print(f"Save {output_filepath}")
+    df.to_csv(output_filepath, index=False, compression='gzip')
 
     return output_filepath
+
+def add_datetime_and_groupby_week(df):
+    df['date_deces'] = pd.to_datetime(df['date_deces'])
+    df['year'] = df['date_deces'].dt.year
+    df['weeknumber'] = df['date_deces'].dt.week
+    df = df.groupby(["year", "weeknumber", "age_bin", "sexe", "departement", "region"], as_index=False)["nb_deces"].sum()
+    return(df)
+
+def historical_data_as_weekly(outputdir="./preprocessed_data/"):
```

Comment:
i feel weird that all `filename` are hardcoded, but it is ok ^^(we need to add filenames in parameters if the function will be use of other purposes).

It should it better to check if it is working properly in tests/ but it is not really important